bin/build.sh

```diff
@@ -322,6 +332,9 @@ fi
 if [ "${PORT_ROOT}x" = "x" ]; then
   printError "PORT_ROOT needs to be defined to the root directory of the tool being ported"
 fi
+if ! [ -d "${PORT_ROOT}" ]; then
```

Comment:
Is this intended to be the same check as https://github.com/ZOSOpenTools/utils/pull/7/files#diff-00bc17178690831bc7abb033a0dede7721a41aebbd076ebd849d98cbca52e962R338?

Reply:
yes. do you want this collapsed into one check instead of 2, or do you see an issue?